sirbot/pythondev/slack.py

```diff
@@ -454,3 +457,20 @@ def add(self, slack):
             message.text = message.text.format(number=len(members),
                                                user=event['user']['id'])
             await slack.send(message)
+
+    async def move_to(self, command, slack, registry):
+        response = command.response()
+        to_id = command.text
```

Comment:
you should rename `to_id` to something more explicit. It's not the `id` but the `name`